guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,45 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You can download and install additional community-maintained modules from Galaxy server.
+For more information, see https://galaxy.ansible.com/docs/using/installing.html[Installing content] in _Ansible Galaxy Documentation_.
+ifdef::satellite[]
++
+Note that {Team} does not provide support for this content.
+endif::[]
+ifdef::satellite[]
+* If you have a subscription for {Team} Automation Hub, you can configure your `ansible-galaxy` to talk to Automation Hub server and download content from there.
+That content is supported by {Team}.
+For more information on configuring Automation Hub connection for `ansible-galaxy`, see
+https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/getting_started_with_automation_hub/proc-configure-automation-hub-server
+endif::[]
+* You rewrite your templates, roles, etc.
+ifdef::satellite[]
+Note that {Team} does not provide support for content that you maintain yourself.
+endif::[]
+
+////
+// Not packaged with RHEL yet
```

Comment:
Maybe tell users to install from pypi? https://pypi.org/project/ansible-lint/

I think this is indeed a very valuable information, so maybe add `ifndef::satellite[]` or tell users to lint on their local machine? Or maybe using a container?